style.css

```diff
@@ -28,15 +23,621 @@ ul {
+}
+
+.pick p {
+  font-size: 1rem;
+  line-height: 1rem;
+  align-self: center;
+  margin: 0;
+  margin-left: 5px;
+  color: black;
+}
+
+.card p {
+  font-size: 1.1rem;
+  line-height: 1.3rem;
+}
+
+.pick p:hover {
+  text-decoration: underline;
+}
+
+.helper {
+  content: "";
+  width: 250px;
+}
+
+.inflation {
+  position: relative;
+  width: 945px;
+  margin: 0 auto;
+  text-align: left;
+}
+
+.img-relative {
+  position: relative;
+}
+
+.boxes {
+  margin-top: 80px;
+  position: absolute;
+  top: 1px;
+  left: 1px;
+  z-index: 10;
+}
+
+.grid-helper {
+  margin-top: 130px;
+  display: grid;
+  grid-template-columns: 1fr 1fr 1fr 1fr;
+  gap: 20px;
+}
+
+.l-font {
+  font-size: 0.8rem;
+  line-height: 1.2rem;
+  text-align: left;
+  font-family: 'Segoe UI', Tahoma, Geneva, Verdana, sans-serif;
+}
+
+.align-left {
+  margin: 15px auto 15px 0;
+  background-color: black;
+}
+
+.s-font {
+  font-size: 0.7rem;
+  line-height: 1.2rem;
+  text-align: left;
+  font-family: 'Segoe UI', Tahoma, Geneva, Verdana, sans-serif;
+}
+
+article .links {
+  display: flex;
+}
+
+#comments-btn {
+  width: 50%;
+  height: 2.5rem;
+}
+
+.links .social-networks {
+  margin-top: 0;
+  flex-grow: 1;
+  display: flex;
+  justify-content: end;
+}
+
+.news {
+  display: flex;
+  padding: 10px;
+  margin-top: 20px;
+  margin-bottom: 1px;
+  padding-bottom: 50px;
+  text-align: left;
+  justify-content: space-between;
+  border-bottom: lightgray 2px solid;
+}
+
+.box-news {
+  width: 71%;
+  border-top: black 1px solid;
+}
+
+.newsletters {
+  display: flex;
+  flex-direction: column;
+  padding-top: 20px;
+}
+
+.newsletters-top {
+  display: flex;
+  justify-content: space-between;
+  font-family: 'Gill Sans', 'Gill Sans MT', Calibri, 'Trebuchet MS', sans-serif;
+}
+
+.most-popular {
+  width: 27%;
+  border-top: black 1px solid;
+  padding-top: 20px;
+}
+
+.cards {
+  display: flex;
+  padding-top: 20px;
+  justify-content: space-between;
+}
+
+.card {
+  position: relative;
+  border: lightgray solid 1px;
+  width: 240px;
+  text-align: left;
+  padding: 120px 10px 30px 10px;
+}
+
+.card img {
+  width: 80px;
+  height: auto;
+  position: absolute;
+  top: 20px;
+  left: 20px;
+}
+
+.top-right {
+  position: absolute;
+  top: 20px;
+  right: 20px;
+  width: 35px;
+  height: 35px;
+  border-radius: 50%;
+  border: black solid 1px;
+  color: black;
+  padding: 9px;
+}
+
+.top-right:hover {
+  background: black;
+  color: white;
+  transition: ease 0.6s;
+}
+
+.card .l-font {
+  font-size: 0.8rem;
+  margin: 0;
+}
+
+.card .latest {
+  position: absolute;
+  bottom: 15px;
+  left: 10px;
+  color: #326891;
+}
+
+.latest:hover {
+  text-decoration: underline;
+}
+
+.more {
+  border-top: black 1px solid;
+  padding-top: 30px;
+  margin-top: 30px;
+  text-align: left;
+}
+
+.more h4 {
+  font-family: 'Gill Sans', 'Gill Sans MT', Calibri, 'Trebuchet MS', sans-serif;
+}
+
+.news-boxes {
+  display: grid;
+  grid-template-columns: 1fr 1fr 1fr;
+  gap: 10px;
+}
+
+.box-img {
```

Comment:
### Aim to remain DRY
- [ ] This rule seems to be repeating more than once and makes you appear as if you are repeating yourself. Do you mind merging these ones so as to make the codebase look neat?

**The repeating rules are on lines `#38` & `#546`**